gui/src/components/onboarding/pages/CalibrationTutorial.tsx

```diff
@@ -32,42 +34,62 @@ export function CalibrationTutorialPage() {
     CalibrationStatus.WAITING
   );
   const [skipButton, setSkipButton] = useState(false);
+  const [settled, setSettled] = useState(false);
   const { timer, isCounting, startCountdown, abortCountdown } = useCountdown({
-    duration: IMU_CALIBRATION_TIME,
-    onCountdownEnd: () => setCalibrationStatus(CalibrationStatus.SUCCESS),
+    duration: settled ? IMU_CALIBRATION_TIME : IMU_SETTLE_TIME,
+    onCountdownEnd: () =>
+      settled
+        ? setCalibrationStatus(CalibrationStatus.SUCCESS)
+        : setSettled(true),
   });
   useTimeout(() => setSkipButton(true), 10000);
   const connectedIMUTrackers = useAtomValue(connectedIMUTrackersAtom);
   const restCalibrationTrackers =
     useRestCalibrationTrackers(connectedIMUTrackers);
   const [rested, setRested] = useState(false);
-  const lastValueMap = useRef(new Map<number, Vector3[]>());
+  const lastValueMap = useRef(new Map<number, Vector3>());
   useEffect(() => {
     const accelLength = restCalibrationTrackers.every((x) => {
       if (
+        x.device?.id?.id === undefined ||
         x.tracker.trackerId?.trackerNum === undefined ||
         x.tracker.trackerId.deviceId?.id === undefined ||
         !x.tracker.linearAcceleration
       )
         return false;
 
-      const trackerId =
-        x.tracker.trackerId.trackerNum + (x.tracker.trackerId.trackerNum << 8);
-      const lastValues = lastValueMap.current.get(trackerId) ?? [];
-      lastValueMap.current.set(trackerId, lastValues);
+      const trackerId = x.device.id.id;
```

Comment:
this breaks extensions as its no longer using `trackerNum`

i see that it previously had a bug of using `trackerNum` two times? probably meant to have deviceId in the bitwise operation

Reply:
it seems like it works correctly anyway, but i will change it to account for trackerNum
deviceId is the same as device.id, i will use the latter because i think it is a little clearer what it represents

Reply:
yeah thats okay